_data/tools.yml

```diff
@@ -2178,4 +2178,12 @@
     Comes complete with high and low-level APIs, diff engine, index and resolver.
   v2: true
   v3: true
-  v3_1: true
\ No newline at end of file
+  v3_1: true
```

Comment:
This should be false as the underlying OpenAPI.NET library that parses the OpenAPI doesn't support 3.1 yet.  We're working on it though :-)

Reply:
Hi @darrelmiller, thanks for the review.
This change was only about adding a line break in the tool above (libopenapi).
OpenAPI-Comparator is indeed only available for version 3.0.